components/org.wso2.carbon.identity.recovery.endpoint/src/main/resources/api.identity.recovery.yaml

```diff
@@ -47,17 +47,17 @@ consumes:
 # The media type(s) consumable by the requestor is specified in the Accept header field
 # of the corresponding request. 
 # The actual media type returned will be specfied in the Content-Type header field 
-# of the of the response.
-# The default of the Accept header is json, i.e. there is not needed to
+# of the response.
+# The default of the Accept header is JSON, i.e., it is not needed to
 # set the value, but supporting it serves extensibility.
 produces: 
   - application/json
 paths:
-  # Endpoint uses to send recovery notificataion
+  # Endpoint used to send recovery notification
```

Comment:
Isn't  this "the endpoint that used to send  recovery notification"

Reply:
Fixed.